Parsers/ASimAuditEvent/Tests/VMwareCarbonBlackCloud_ASimAuditEvent_DataTest.csv

```diff
@@ -0,0 +1,8 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 100922 records (100.0%) for field [EventProduct] of type [Enumerated]: [""Carbon Black Cloud""] (Schema:AuditEvent)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 100922 records (100.0%) for field [EventVendor] of type [Enumerated]: [""VMware""] (Schema:AuditEvent)"
+"(2) Info: Empty value in 100717 records (99.8%)  in recommended field [NewValue] (Schema:AuditEvent)"
+"(2) Info: Empty value in 100733 records (99.81%)  in optional field [TargetDvcId] (Schema:AuditEvent)"
+"(2) Info: Empty value in 100906 records (99.98%)  in optional field [OldValue] (Schema:AuditEvent)"
```

Comment:
Both Old and New Values are 99% missing. We should have these fields in an 'Audit' log